include/_pia.h

```diff
@@ -34,18 +39,45 @@
 #define __PIA_H
 
 
-/* Define a structure with the pia register offsets */
+/* Define a structure with the PIA register offsets */
 struct __pia {
     unsigned char   porta;  /* port A data r/w */
+/* Paddle 0-3 triggers (per PORTA bits) */
+#define PORTA_PTRIG3 0x80
+#define PORTA_PTRIG2 0x40
+#define PORTA_PTRIG1 0x08
+#define PORTA_PTRIG0 0x04
+
+
     unsigned char   portb;  /* port B data r/w */
+/* Paddle 4-7 triggers (per PORTB bits); only 400/800 had four controller ports */
+#define PORTB_PTRIG7 0x80
+#define PORTB_PTRIG6 0x40
+#define PORTB_PTRIG5 0x08
+#define PORTB_PTRIG4 0x04
+
+    /* See also: "JOY_xxx_MASK" in "atari.h" */
```

Comment:
As far as I can see you didn't indent comment-only lines in the other files. So please don't do here too.